man/man1/pkcsstats.1.in

```diff
@@ -0,0 +1,135 @@
+.\" pkcsstats.1
+.\"
+.\" Copyright IBM Corp. 2021
+.\" See LICENSE for details.
+.\"
+.TH PKCSSTATS 1 "October 2021" "@PACKAGE_VERSION@" "openCryptoki"
+.SH NAME
+pkcsstats \- utility to display mechanism usage statistics for openCryptoki.
+
+.SH SYNOPSIS
+.B pkcsstats
+.RB [ OPTIONS ]
+.
+.PP
+.B pkcsstats
+.BR \-\-help | \-h
+.br
+
+.SH DESCRIPTION
+Displays mechanism usage statistics for openCryptoki. Usage statistics are
+collected by openCryptoki on a per user basis. For each user, mechanism
+usage is counted per configured slot and mechanism. For each mechanism a set of
+counters exist, one for each cryptographics strength of the cryptographic
+key used with the mechanism.
+.PP
+The available strengths are defined in the strength configuration file
+\fB/etc/opencryptoki/strength.conf\fP. Supported strengths are 112, 128, 192,
+and 256 representing the corresponding strength in bits.
+The strength configuration file defines how the strength is determined for the
+various key types. A strength of zero is used to count those mechanisms that
+do not use a key, or where the key strength is less than 112 bits.
+.PP
+.B Note:
+The strength does not specify the cryptographic strength of the mechanism, but
+the cryptographic strength of the key used with the mechanism (if any).
+For example, usage of mechanism CKM\_SHA256 is reported under strength 0,
+because no key is used with this mechanism. However, usage of mechanism
+CKM\_AES\_CBC is reported under strength 128, 192, or 256, dependent on the
```

Comment:
Technically, if a strength.json maps all symmetric keys to strength 0 (don't know why someone would do that, but it is possible), CKM_AES_CBC would end up in strength 0 bucket.

Reply:
Right, this was mean with the part in the bracket: 
_...dependent on the cryptographic size of the AES key used with it (**and the definitions in the strength configuration file**)_